dolphinscheduler-task-plugin/dolphinscheduler-task-dataquality/src/main/java/org/apache/dolphinscheduler/plugin/task/dq/DataQualityTask.java

```diff
@@ -71,6 +71,16 @@ public class DataQualityTask extends AbstractYarnTask {
      */
     private static final String SPARK_COMMAND = "${SPARK_HOME}/bin/spark-submit";
 
+    /**
+     * --class CLASS_NAME
+     */
+    public static final String MAIN_CLASS = "--class";
+
+    /**
+     * DataQuality mainClass name
+     */
+    private static final String DATA_QUALITY_MAIN_CLASS_NAME = "org.apache.dolphinscheduler.data.quality.DataQualityApplication";
```

Comment:
We don't need to add this after add manifest in pom.xml

Reply:
Yes, you're right. thx